ios/Helio.xcodeproj/project.pbxproj

```diff
@@ -1395,7 +1395,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "export NODE_BINARY=node\n../node_modules/react-native/scripts/react-native-xcode.sh\n";
+			shellScript = "export NODE_OPTIONS=--openssl-legacy-provider\nexport NODE_BINARY=node\n../node_modules/react-native/scripts/react-native-xcode.sh\n";
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Same as above, the addition of `NODE_OPTIONS=--openssl-legacy-provider` is a workaround for the Node.js issue where the default OpenSSL library was changed in Node.js 17.0.0. This change should be removed once all dependencies are updated to work with the new OpenSSL library.
```diff
- export NODE_OPTIONS=--openssl-legacy-provider
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->